package com.peterso.fetch_application.exceptions;

public class IdNotFoundException extends RuntimeException {

  public IdNotFoundException(String message, Throwable cause) {
    super(message, cause);
  }
}
